src/components/SubHeader/Restaurant/Offers/components/Offer.js

```diff
@@ -0,0 +1,33 @@
+// prop-types
+import PropTypes from "prop-types";
+
+// reader
+import offerReader from "../readers/OfferReader";
+
+// icons
+import offerIcon from "../../../../../assets/icons/offer.svg";
+
+// css
+import "./Offer.css";
+
+const Offer = function ({ item }) {
+  const discount = offerReader.discount(item);
+  const coupon = offerReader.coupon(item);
+  return (
+    <div className="offer flex">
+      <img src={offerIcon} className="icon" alt={coupon} />
```

Comment:
try to use IconWithLabel

Reply:
> try to use IconWithLabel
I've to do many tweaks inside css to make it look alike if I would use IconWithLabel